yarn.lock

```diff
@@ -9444,11 +9450,6 @@ caniuse-api@^3.0.0:
     lodash.memoize "^4.1.2"
     lodash.uniq "^4.5.0"
 
-caniuse-db@1.0.30000772:
```

Comment:
I think this should be not removed.

Reply:
Hmm thats strange, I didn't remove that?

Reply:
What I mean is this is something yarn must have done as I just did __`yarn add @types/jest@26.0.0`__ in all packages folders. But I'll double-check again.